include/deal.II/base/thread_local_storage.h

```diff
@@ -246,29 +272,12 @@ namespace Threads
   }
 
 
-  template <typename T>
-  inline
-#    ifdef DEAL_II_WITH_THREADS
-    tbb::enumerable_thread_specific<T> &
-#    else
-    T &
-#    endif
-    ThreadLocalStorage<T>::get_implementation()
```

Comment:
do you want to mention this as an incompatibility in the changelog?

Reply:
Yes. Done. :smile: